test/unit/specs/stores/active-media-store.spec.js

```diff
@@ -0,0 +1,56 @@
+import { setActivePinia, createPinia } from 'pinia'
+
+import { AUDIO } from '~/constants/media'
+
+import { useActiveMediaStore } from '~/stores/active-media'
+
+const initialState = { type: null, id: null, status: 'ejected', message: null }
+const statuses = ['ejected', 'paused', 'playing']
+
+describe('Active Media Store', () => {
+  beforeEach(() => {
+    setActivePinia(createPinia())
```

Comment:
Does this have the effect of resetting the store before each test?

Reply:
Yes:
// creates a fresh pinia and make it active so it's automatically picked
// up by any useStore() call without having to pass it to it:
// `useStore(pinia)`